src/ontology/mondo-ingest.Makefile

```diff
@@ -519,6 +519,24 @@ $(REPORTDIR)/%_mapped_deprecated_terms.robot.template.tsv: $(REPORTDIR)/%_mappin
 	--mapping-status-path $(REPORTDIR)/$*_mapping_status.tsv \
 	--outpath $@
 
+####################################
+### QC / test ######################
```

Comment:
## QC section in makefile
This setup will be changed when #319 is decided on and implemented, and these checks will go into that setup.